app/src/main/res/values/styles.xml

```diff
@@ -18,4 +19,14 @@
     <style name="SocialAccountButton" parent="Widget.AppCompat.ImageButton">
         <item name="android:background">@drawable/social_account_shape</item>
     </style>
+
+    <style name="ExpandedToolBar" parent="TextAppearance.Design.CollapsingToolbar.Expanded">
+        <item name="android:textSize">34sp</item>
```

Comment:
Use values from dimens.xml file instead of hardcoded one. You should avoid hardcoded values even in styles.xml file. The same comment for `CollapsingToolbar` style

Reply:
fixed